pkg/webui/console/components/gateway-data-form/index.js

```diff
@@ -148,7 +166,30 @@ class GatewayDataForm extends React.Component {
           placeholder={sharedMessages.gatewayEUI}
           component={Input}
           tooltipId={tooltipIds.GATEWAY_EUI}
+          onBlur={this.onDisplayClaimChange}
         />
+        {shouldDisplayClaimWarning && (
```

Comment:
I dont think we need this in OS since there is no claiming view. Basiclaly, `console/gateways/claim` is not reachable.